<launch>
  <param name="robot_description" command="$(find xacro)/xacro.py $(find tr3_description)/robots/tr3.urdf.xacro" />

  <include file="$(find tr3_moveit)/launch/planning_context.launch">
    <arg name="load_robot_description" value="true"/>
  </include>

  <node pkg="tr3_moveit" name="forward_ik" type="forward_ik_server" />
  <node pkg="tr3_moveit" name="inverse_ik" type="inverse_ik_server" />
</launch>
